src/api/schemas/responses/responses.ts

```diff
@@ -178,3 +179,14 @@ export const RunFaucetResponseSchema = Type.Object(
   }
 );
 export type RunFaucetResponse = Static<typeof RunFaucetResponseSchema>;
+
+export const BlockListV2ResponseSchema = Type.Object({
```

Comment:
We should generalize this structure similar to what we did in `PaginatedResponse()` so we have something reusable like `PaginatedCursorResponse()`

Reply:
Done!